build/common.ps1

```diff
@@ -454,20 +454,44 @@ Function Test-CoreProjects {
     $xtests | Test-XProject -Configuration $Configuration
 }
 
+Function Test-MSBuildVersionPresent {
+    [CmdletBinding()]
+    param(
+        [parameter(Mandatory=$True, position=0)]
+        [string]$MSBuildVersion
+    )
+    
+   	$MSBuildExe = Get-MSBuildExe $MSBuildVersion
+
+    Test-Path $MSBuildExe
+}
+
+Function Get-MSBuildExe {
+    param(
+        [parameter(Mandatory=$True, position=0)]
+        [string]$MSBuildVersion
+    )
+    
+    $MSBuildExe = Join-Path $MSBuildRoot ($MSBuildVersion + ".0")
+    Join-Path $MSBuildExe $MSBuildExeRelPath
+}
+
 Function Build-ClientsProjects {
     [CmdletBinding()]
     param(
         [string]$Configuration = $DefaultConfiguration,
         [string]$ReleaseLabel = $DefaultReleaseLabel,
         [int]$BuildNumber = (Get-BuildNumber),
+        [parameter(Mandatory=$True)]
+        [string]$MSBuildVersion,
```

Comment:
Default values are given here for every other parameter (all of them are important) as it was a convention. Default value for msbuild version would provide us with backcompat and convenience.

If you want to enforce strict parameter verification policy here consider adding `ValidateSet`.